src/mlpack/tests/main_tests/extra_trees_test.cpp

```diff
@@ -0,0 +1,185 @@
+/**
+ * @file tests/main_tests/random_forest_test.cpp
```

Comment:
Don't forget to add this to `CMakeLists.txt` in `src/mlpack/tests/main_tests/CMakeLists.txt`. :)  (Also, don't forget to update the comment too!)

Reply:
This comment still applies---these tests won't be run unless you add it to `CMakeLists.txt`.